network/network_windows.go

```diff
@@ -185,6 +190,55 @@ func (nm *networkManager) newNetworkImplHnsV1(nwInfo *NetworkInfo, extIf *extern
 	return nw, nil
 }
 
+// add ipv4 and ipv6 (if dual stack mode) to windows Node
+func (nm *networkManager) addNewNetRulesToWindowsNode(nwInfo *NetworkInfo) error {
+	var (
+		err error
+		out string
+	)
+
+	// get interface name of VM adapter
+	ifName := nwInfo.MasterIfName
+	if !strings.Contains(nwInfo.MasterIfName, ifNamePrefix) {
+		ifName = fmt.Sprintf("%s (%s)", ifNamePrefix, nwInfo.MasterIfName)
+	}
+
+	// add ipv4 and ipv6 new net rules to windows node
+	for _, subnet := range nwInfo.Subnets {
+		prefix := subnet.Prefix.String()
+		gateway := subnet.Gateway.String()
+
+		ip, _, _ := net.ParseCIDR(prefix)
+		if ip.To4() != nil {
+			// netsh interface ipv4 add route $subnetV4 $hostInterfaceAlias "0.0.0.0" metric=270
+			netshV4DefaultRoute := fmt.Sprintf(netRouteCmd, "ipv4", prefix, ifName, ipv4DefaultHop, "270")
+			if out, err = nm.plClient.ExecuteCommand(netshV4DefaultRoute); err != nil {
+				log.Printf("[net] Adding ipv4 default route failed: %v:%v", out, err)
```

Comment:
should we be returning err here rather than proceed forward?
If this route is not set successfully, we will get datapath failure right?

same comment for ipv6 below